R/github_token.R

```diff
@@ -294,3 +296,48 @@ scold_for_scopes <- function(scopes) {
   )
   ui_bullets(message)
 }
+
+#' Lock and unlock a branch on GitHub
+#'
+#' These functions lock and unlock a branch on GitHub so that it's not possible
+#' for anyone to make any changes. This is used as part of the release process
+#' to ensure that you don't accidentally merge any pull requests or push any
+#' commits while you are waiting for CRAN to get back to you.
```

Comment:
Anything about permissions required to perform this action?

Reply:
You need to be an admin or owner, which I've documented. It looks like this is already in line with our governance tidyup where the maintainer has admin access: https://github.com/tidyverse/tidyups/blob/main/004-governance.md#maintainer.